modules/gsp-cluster/data/values.yaml

```diff
@@ -119,6 +119,9 @@ serviceOperator:
 RDSFromWorkerSecurityGroup: ${rds_from_worker_security_group}
 privateDBSubnetGroup: ${private_db_subnet_group}
 
+redisFromWorkerSecurityGroup: ${redis_from_worker_security_group}
```

Comment:
I feel like I'm too late on this one, but this naming is weird.  A security group identifies a thing, not a relationship.

Again, not a blocker for this PR, but if it's not hard to rename all the things I'd prefer this be just `redisSecurityGroup` (and similarly `rdsSecurityGroup` elsewhere)